From ee0fbddd73a5160e935ddf12ccdaba740ba4838e Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 11 Mar 2021 16:48:28 -0800
Subject: [PATCH 1/2] Bundle Asio on RHEL 8

Asio is not currently available for RHEL 8, but it appears that Fast-DDS
is able to bundle it. Rather than introduce an Asio package to the
bootstrap repository just for this package, we can flip this flag to
bundle it.

The CMake logic in Fast-DDS expects the sources to be in a git
repository, and uses submodules to download the bundled sources. When
the package is bloomed, the submodules are initialized and included in
the tarball, but the sources are no longer part of a git repository.
Since we know that the directory has already been populated, we need to
suppress the submodule initialization and update step.

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 package.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/package.xml b/package.xml
index a897fe54..b56467c5 100644
--- a/package.xml
+++ b/package.xml
@@ -14,8 +14,6 @@
   <url type="bugtracker">https://github.com/eProsima/Fast-DDS/issues</url>
   <url type="repository">https://github.com/eProsima/Fast-DDS</url>
 
-  <build_depend>asio</build_depend>
-
   <depend>fastcdr</depend>
   <depend>foonathan_memory_vendor</depend>
   <depend>libssl-dev</depend>
-- 
2.31.1

